projects/BFDR/NatalityRecord_submissionProperties.cs

```diff
@@ -8412,7 +8417,7 @@ public Dictionary<string, string> PayorTypeFinancialClass
         /// <para>Console.WriteLine($"Child's Place Of Birth Type: {ExampleBirthRecord.PayorTypeFinancialClassHelper}");</para>
         /// </example>
         [Property("PayorTypeFinancialClassHelper", Property.Types.String, "PayorTypeFinancialClassHelper", "Principal source of Payment for this delivery Helper.", false, IGURL.CoveragePrincipalPayerDelivery, true, 4)]
-        [FHIRPath("Bundle.entry.resource.where($this is Coverage).where(meta.profile == " + ProfileURL.CoveragePrincipalPayerDelivery + ")", "")]
+        [FHIRPath("Bundle.entry.resource.where($this is Coverage)", "")]
```

Comment:
is coverage unique so this fhir path can still identify this component without the meta profile?